__checks__/dev.test.js

```diff
@@ -1,15 +1,14 @@
 const fs = require("fs");
-const { argv } = require("yargs");
+const { path, org } = require('../src/argv');
 
 describe("Development", () => {
-  let { path: rootPath, org } = argv;
-  let pkg = JSON.parse(fs.readFileSync(`${rootPath}/package.json`, "utf8"));
+  let pkg = JSON.parse(fs.readFileSync(`${path}/package.json`, "utf8"));
```

Comment:
move this into a `beforeAll` hook to ensure it runs at test time rather than require-time

Reply:
oh, sorry, it's unrelated to your change